src/include/txn.i

```diff
@@ -350,15 +351,17 @@ __wt_txn_modify(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, WT_UPDATE *upd)
 	upd->txnid = session->txn.id;
 
 #ifdef HAVE_TIMESTAMPS
-	__wt_txn_update_set_timestamp(session, op, false, NULL);
+	__wt_txn_update_set_timestamp(session, op, false, &timestamp_set);
```

Comment:
@bvpvamsikrishna the intent makes sense to me - I'm proposing to remove the `timestamp_set` return from `__wt_txn_update_set_timestamp` in WT-4300 - is there something else we could use to determine if this optimization is OK?